ui/pages/confirmations/components/confirm/info/approve/edit-spending-cap-modal/edit-spending-cap-modal.tsx

```diff
@@ -53,9 +53,10 @@ export const EditSpendingCapModal = ({
     transactionMeta.txParams.data,
   );
 
-  const accountBalance = Number(
-    calcTokenAmount(userBalance || '0', Number(decimals || '0')),
-  );
+  const accountBalance = calcTokenAmount(
+    userBalance ?? '0',
+    decimals,
```

Comment:
```suggestion
    decimals ?? '0',
```
calcTokenAmount does not currently handle if decimals == null. Not sure if we should be concerned but maybe just in case?

Reply:
I added the Number coercion as well because calcTokenAmount expects decimals as a number